server/__init__.py

```diff
@@ -339,7 +339,8 @@ def create_app(nl_root=DEFAULT_NL_ROOT):
       secret_name = secret_client.secret_version_path(cfg.SECRET_PROJECT,
                                                       'mixer-api-key', 'latest')
       secret_response = secret_client.access_secret_version(name=secret_name)
-      app.config['DC_API_KEY'] = secret_response.payload.data.decode('UTF-8')
+      app.config['DC_API_KEY'] = secret_response.payload.data.decode(
+          'UTF-8').replace('\n', '')
```

Comment:
I found I needed to strip a trailing `\n` in order for the API key to parse correctly, but let me know if there's a better way to do this.